src/vcf/validator.cpp

```diff
@@ -145,6 +145,16 @@ namespace ebi
         }
 
         validator.end();
+        for (auto &error : validator.errors()) {
```

Comment:
These loops shouldn't be necessary, `validator.end()` is just a formality to clean up the status at the end but won't check any extra characters.